### PR TITLE
Status page groups

### PR DESCRIPTION
**Pull request by CarwynNelson**

### CODE REVIEW

File: examples/basic/main.tf [hunk 1]
```diff
@@ -7,11 +7,17 @@ resource "random_id" "status_page_subdomain" {
   prefix      = "tf-status-"
 }
 
+resource "betteruptime_status_page_group" "this" {
+  name = "tf group"
```
Comment (PetrHeinz) [inline examples/basic/main.tf]: 💄 
```suggestion
  name = "Status pages from Terraform"
```

File: internal/provider/resource_status_page_group.go [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+		Description: "The ID of this status page group.",
+		Type:        schema.TypeString,
+		Optional:    false,
+		Computed:    true,
+	},
+	"name": {
+		Description: "A name of the group that you can see in the dashboard.",
+		Type:        schema.TypeString,
+		Optional:    true,
+		Computed:    true,
```
Comment (PetrHeinz) [inline internal/provider/resource_status_page_group.go]: I couldn't find the API docs near https://betterstack.com/docs/uptime/api/list-all-existing-status-pages/, but I have a feeling the name should be required, shouldn't it?

```suggestion
		Required:    true,
```

**Review by PetrHeinz — APPROVED** [review]
Thank you @CarwynNelson for adding that 🙌 

Could you just please double check the name parameter? ResourceGroup seem to validate the presence of name from what I see in the API controller code. If true, maybe we should update the other group resource... What do you think?

Feel free to merge and release as new patch version, assuming all CI check pass 👍

File: internal/provider/resource_status_page_group.go (hunk 2 above)
Comment (CarwynNelson) [reply internal/provider/resource_status_page_group.go]: Good catch

**Pull request merged**